en/docs/cli/ls.md

```diff
@@ -4,18 +4,32 @@ guide: docs_cli
 layout: guide
 ---
 
-<p class="lead">List installed packages.</p>
+<p class="lead">List all dependencies installed for the current working directory</p>
```

Comment:
I don't think you need to specify current working directory in the description of this command.